copier.yml

```
@@ -411,6 +411,17 @@ smtp_relay_password:
   help: >-
     What is your SMTP password?
 
+smtp_relay_version:
+  type: str
+  when: *has_smtp
+  default: "8.0"
```

Let's update the default 😃 

```suggestion
  default: "9.0"
```